### ISSUE TITLE
 is this project is active and new versions expected ?

### ISSUE THREAD

**Issue by etombi** [alixaxel/chrome-aws-lambda#279]
hi , guys, and thank you for your great work, i saw that it wasn't updated for some time , 
are any updates are expected?

**Comment by hasan-wajahat** [comment]
@etombi From my understanding this project is dead. The owner has not responded to anything. However @sparticuz has been kind enough to create his own fork which seems to be working fine for us: https://github.com/Sparticuz/chrome-aws-lambda. (there are some memory concerns which hopefully will be resolved)

**Comment by bitconym** [comment]
@alixaxel is it true that the project is abandoned, can you update the readme of deprecation or even better give the project to someone to maintain it? Maybe @Sparticuz will be interested.

**Comment by mstoyanovv** [comment]
I understand that the project is dead but is it even working as expected at this point? I get errors with this package and with the one from Sparticuz..

**Comment by hasan-wajahat** [comment]
@mstoyanovv what was the error?

**Comment by mstoyanovv** [comment]
@hasan-wajahat just fixed it by making the puppeteer version locked to 10.0.0

**Comment by hasan-wajahat** [comment]
Yeah I thought so, I was able to do the same by locking it to a specific version that worked for me.

**Comment by bitconym** [comment]
Yes, @mstoyanovv make sure to lock the puppeteer version per this list: https://github.com/Sparticuz/chrome-aws-lambda#versioning.

For example, if you install `@sparticuz/chrome-aws-lambda@~14.4.1` then you need `puppeteer` (or `puppeteer-core`) version `14.4.*`
